### PR TITLE
Remove unknown plugins and GoobiHotfolder stuff

### PR DESCRIPTION
**Pull request by matthias-ronge**
Both "command" and "webapi" plugin types are not documented and not backed with open source plug-ins. Same is true for GoobiHotfolder stuff. This is to remove unused code.

### CODE REVIEW

**Comment by matthias-ronge** [comment]
@henning-gerhardt please add a reviewer

File: Kitodo/src/main/java/org/goobi/production/flow/jobs/AbstractGoobiJob.java [hunk 1]
```diff
@@ -1,91 +0,0 @@
...
- *
- * For the full copyright and license information, please read the
- * GPL3-License.txt file that was distributed with this source code.
- */
-
-package org.goobi.production.flow.jobs;
-
-import org.apache.logging.log4j.LogManager;
-import org.apache.logging.log4j.Logger;
-import org.quartz.Job;
```
Comment (henning-gerhardt) [inline Kitodo/src/main/java/org/goobi/production/flow/jobs/AbstractGoobiJob.java]: As you remove last usage of `org.quartz` library the dependency in `Kitodo/pom.xml` and file `Kitodo/src/main/resources/quartz.properties` should be removed.

Comment (matthias-ronge) [reply Kitodo/src/main/java/org/goobi/production/flow/jobs/AbstractGoobiJob.java]: Good catch 👍

**Comment by matthias-ronge** [comment]
Quartz removed: Less sand in the gears. ;-)

**Pull request merged**